crossplane.yaml

```diff
@@ -0,0 +1,95 @@
+spec:
+  crossplane:
+    version: ">=v1.0.0-0"
+  dependsOn:
+    - provider: registry.upbound.io/crossplane/provider-azure
+      version: ">=v0.18.1-0"
+    - provider: registry.upbound.io/crossplane/provider-helm
+      version: ">=v0.3.6-0"
```

Comment:
latest provider-helm is v0.9.0, so you could set this to that: https://github.com/crossplane-contrib/provider-helm/releases/tag/v0.9.0